composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/browser-kit": "^6.3 || ^7.0",
         "symfony/filesystem": "^6.3 || ^7.0",
         "symfony/framework-bundle": "^6.3.5 || ^7.0",
-        "symfony/phpunit-bridge": "^6.3 || ^7.0",
+        "symfony/phpunit-bridge": "~6.3.10 || ^6.4.1 || ^7.0.1",
```

Comment:
Is there a specific reason to user `~` over `^`?

Reply:
We don't want 6.4.0 to be installed.

Reply:
https://getcomposer.org/doc/articles/versions.md#next-significant-release-operators

Reply:
I agree, but then the second kicks in, np?

It would be simpler to read like this:
`^6.3.10 || ^7.0.1`

+

adding a conflict for `=6.4.0`